src/assets/wise5/components/match/match-student/match-student.component.ts

```diff
@@ -611,15 +618,40 @@ export class MatchStudent extends ComponentStudent {
     return componentState;
   }
 
-  removeStatusFromBucketItems(buckets: any): any {
+  cleanBuckets(buckets: any): any {
     for (const bucket of buckets) {
+      bucket.value = this.getCleanedValue(bucket);
       for (const item of bucket.items) {
+        item.value = this.getCleanedValue(item);
         delete item.status;
       }
     }
     return buckets;
   }
 
+  getCleanedValue(matchObj: any): string {
+    const value = this.getValueById(this.originalComponentContent, matchObj.id);
+    if (value == null) {
+      return matchObj.value;
+    } else {
+      return value;
+    }
```

Comment:
Use nullish coalescing for brevity?
```suggestion
    return this.getValueById(this.originalComponentContent, matchObj.id) ?? matchObj.value;
```